### PR TITLE
Added Transaction details page.

### PR DESCRIPTION
**Pull request by ndemidovich-sc**

### CODE REVIEW

**Comment by ndemidovich-sc** [comment]
I think We can resolve problem with displaying recipients addresses later.

File: src/routes/Transaction/TransactionOverview.tsx [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+  {
+    render: (tx: Transaction) =>
+      tx?.to?.length > 1
+        ? tx?.to?.length + ' Addresses'
+        : tx?.to?.length === 1 && (
+            <CopyValueToClipboard
+              label={truncateHash(tx.to[0] || '', 2, 4)}
+              value={tx.to[0] || ''}
+              copyTooltipText={'Copy address to'}
+              copiedTooltipText={'Address copied'}
```
Comment (azhurauski-sc) [inline src/routes/Transaction/TransactionOverview.tsx:50]: color of the copy button is not correct
```
      iconButtonProps={{
        color: NAMED_COLORS.GREY,
      }}
```

File: src/routes/Transaction/TransactionOverview.tsx [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+    label: 'Memo',
+    icon: SizeIcon,
+  },
+  {
+    render: (tx: Transaction) => (
+      <CopyValueToClipboard
+        label={truncateHash(tx?.blockHash || '', 2, 4)}
+        value={tx?.blockHash || ''}
+        copyTooltipText={'Copy block hash'}
+        copiedTooltipText={'Block hash copied'}
```
Comment (azhurauski-sc) [inline src/routes/Transaction/TransactionOverview.tsx:67]: color of the copy button is not correct
```
      iconButtonProps={{
        color: NAMED_COLORS.GREY,
      }}
```

File: src/routes/Transaction/TransactionOverview.tsx [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+    label: 'Block Hash',
+    icon: DifficultyIcon,
+  },
+  {
+    render: (tx: Transaction) => (
+      <CopyValueToClipboard
+        label={truncateHash(tx?.hash || '', 2, 4)}
+        value={tx?.hash || ''}
+        copyTooltipText={'Copy Transaction hash'}
+        copiedTooltipText={'Transaction hash copied'}
```
Comment (azhurauski-sc) [inline src/routes/Transaction/TransactionOverview.tsx:79]: color of the copy button is not correct
```
      iconButtonProps={{
        color: NAMED_COLORS.GREY,
      }}
```

File: src/routes/Transaction/TransactionOverview.tsx [hunk 4]
```diff
@@ -0,0 +1,254 @@
...
+    hash
+  )
+
+  return (
+    <Flex flexDirection="column" pb="0" bg="transparent" w="100%">
+      <Box>
+        <BackButtonLink
+          mb="1rem"
+          to={ROUTES.ACCOUNTS}
+          label={'Back to all accounts'}
```
Comment (azhurauski-sc) [inline src/routes/Transaction/TransactionOverview.tsx:123]: This should return to account overview, not all accounts

File: src/routes/Transaction/TransactionOverview.tsx [hunk 5]
```diff
@@ -0,0 +1,254 @@
...
+                    <Box mr="1rem">
+                      <LargeArrowLeftDown h="1.125rem" w="1.125rem" />
+                    </Box>
+                    <Box
+                      overflow="hidden"
+                      whiteSpace="nowrap"
+                      textOverflow="ellipsis"
+                    >
+                      <chakra.h5>
+                        {truncateHash(spend.nullifier, 2, 4)}
```
Comment (azhurauski-sc) [inline src/routes/Transaction/TransactionOverview.tsx:204]: Copy to clipboard is missing

File: src/svgx/SizeIcon.tsx [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+        fill="#0D0C22"
+      />
+    </Icon>
+  )
+}
+
+export default SizeIcon
```
Comment (azhurauski-sc) [inline src/svgx/SizeIcon.tsx:29]: Do we need to add those icons her? They should be available in the ui-kit.

**Review by azhurauski-sc — CHANGES_REQUESTED** [review]

File: src/svgx/SizeIcon.tsx (hunk 6 above)
Comment (ndemidovich-sc) [reply src/svgx/SizeIcon.tsx:29]: For now there no such icons in UI-Kit. I think we can move them in UI-Kit in separate task and update dependencies in block-explorer and here.

**Review by skylarrichard123 — CHANGES_REQUESTED** [review]
Added the last 2 change requests:
https://www.figma.com/file/fYw3aNgfS0qCS2jezGqv8P/Wallet-(New-Ui-Kit)?node-id=4435%3A29161&t=AbgfF6ecaCp3OO2Z-1

**Comment by ndemidovich-sc** [comment]
> Added the last 2 change requests: https://www.figma.com/file/fYw3aNgfS0qCS2jezGqv8P/Wallet-(New-Ui-Kit)?node-id=4435%3A29161&t=AbgfF6ecaCp3OO2Z-1

Fixed

**Review by skylarrichard123 — CHANGES_REQUESTED** [review]
Screen Shot 2023-01-18 at 10 42 30 AM
Inputs / outputs doesnt need to span the full width, it can remain a 1 column card at this size.

**Review by azhurauski-sc — APPROVED** [review]

**Comment by ndemidovich-sc** [comment]
@skylarrichard123 Fixed

**Review by skylarrichard123 — APPROVED** [review]

**Pull request merged**